From 3c8377909637df1032b2011c9ff5364f48a98d94 Mon Sep 17 00:00:00 2001
From: Steve Kondik <steve@cyngn.com>
Date: Thu, 1 Sep 2016 21:40:27 -0700
Subject: [PATCH 13/39] extract_utils: Fix device online check

 * No need to look for /sbin/recovery as adbd will report
   the device as being in recovery mode already. Also, this
   check appears to hang on N.

Change-Id: I3bcb427835ae3cd37c7ea56cbc57bba0d18fcc04
---
 build/tools/extract_utils.sh | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/build/tools/extract_utils.sh b/build/tools/extract_utils.sh
index e6951d61..8a0b3f28 100644
--- a/build/tools/extract_utils.sh
+++ b/build/tools/extract_utils.sh
@@ -499,8 +499,7 @@ EOF
 # Return success if adb is up and not in recovery
 function _adb_connected {
     {
-        if [[ "$(adb get-state)" == device &&
-              "$(adb shell test -e /sbin/recovery; echo $?)" == 0 ]]
+        if [[ "$(adb get-state)" == device ]]
         then
             return 0
         fi
-- 
2.15.0

